lock-api/src/main/metrics/metric-schema.yml

```diff
@@ -0,0 +1,13 @@
+options:
+  javaPackage: 'com.palantir.atlasdb.timelock.api'
```

Comment:
nit: I'd just use com.palantir.lock.client since that's where the ConjureTimelockServiceBlocking exists